src/App/App.tsx

```diff
@@ -1,9 +1,42 @@
-import {Wrapper} from "./App.styles"
-import Scoreboard from "../components/Scoreboard/Scoreboard";
+import { useState, useEffect } from "react";
+import { Wrapper } from "./App.styles";
+import Scoreboard from "../components/Scoreboard";
+import Options from "../components/Options";
+import RulesBtn from "../components/RulesBtn";
+import Rules from "../components/Rules";
+import Duel from "../components/Duel";
+import useLocalStorage from "./useLocalStorage";
+
 function App() {
+  const [rulesActive, setRulesActive] = useState<boolean>(false);
+  const [playerChoice, setPlayerChoice] = useState<Choice>({} as Choice);
```

Comment:
jeśli nie potrzebujesz tutaj pustego obiektu to możesz po prostu zostawić puste nawiasy `useState<Choice>()` dzięki czemu nie będziesz musiał podawać 'as Choice'